src/config/mod.rs

```diff
@@ -28,16 +28,21 @@ pub const DEFAULT_PROF_NAME: &str = "default";
  All should start with ENV_VAR_PREFIX (CT_xxx).
 ************************************************************************/
 // Default prefix for environment variables added by CloudTruth.
-pub const ENV_VAR_PREFIX: &str = "CT_";
+pub const ENV_VAR_PREFIX: &str = "CLOUDTRUTH_";
 
 // Environment variable name used to specify the CloudTruth API value, so it does not need to be
 // specified on the command line.
 #[allow(dead_code)]
-pub const CT_API_KEY: &str = "CT_API_KEY";
+pub const CT_API_KEY: &str = "CLOUDTRUTH_TOKEN";
+
+// The old environment variable was named 'CT_API_KEY', and we want to provide a better
+// update process.
+#[allow(dead_code)]
+pub const CT_OLD_API_KEY: &str = "CT_API_KEY";
 
 // Environment variable name used to override the default server URL.
 #[allow(dead_code)]
-pub const CT_SERVER_URL: &str = "CT_SERVER_URL";
+pub const CT_SERVER_URL: &str = "CLOUDTRUTH_SERVER_URL";
```

Comment:
An alternate to this would be using CLOUDTRUTH_HOST (and CLOUDTRUTH_PORT, although I don't think this applies here), and not have the customer specify a URI.

Reply:
We can consider that for the future... This is used mainly for testing, and does not even have an option on the command line.